app/Services/Media.php

```diff
@@ -229,12 +229,22 @@ public function getInfo(SplFileInfo $file)
             return;
         }
 
-        $track = array_get($info, 'comments.track_number', [0])[0];
+        $trackIndices = [
+            'comments.track',
+            'comments.tracknumber',
+            'comments.track_number',
+        ];
+        for ($i = 0; $i < count($trackIndices); $i++) {
```

Comment:
You can't do that because if `comments.track` works, then your code doesn't have it breaking and it will proceed to the next index to check which will, if invalid, simply return `0`. There has to be a check and a break when a successful track index is found.